utils/vlog/tests/timing_logic/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_file_target(FILE carry4_vpr.sim.v SCANNER_TYPE verilog)
+
+v2x_test_both(
+  NAME carry4_vpr
+  TOP_MODULE carry4_vpr
+  SDF_FILE CLBLL_L.sdf
+  SDF_CELLS SLICEL/CARRY4 SLICEL/CARRY4_AX SLICEL/CARRY4_BX SLICEL/CARRY4_CX SLICEL/CARRY4_DX
```

Comment:
The way I modelled DIn lines was via the O5 cell, not the AX cells.

This strategy doesn't handled SLICEL vs SLICEM like @kgugala's patch does.  Are you coordinating with @kgugala to avoid duplication of work?

Reply:
You are referring to a test example. It is not generic. In order for the whole SDF timing import to work we need to do significant changes to the architecture and how it is built by CMake.

The approach with timing import through V2X requires us to have a different model for each timing variant of a pb_type.

And yes, I am coordinating with @kgugala

Reply:
Can @kgugala or you explain the relationship between this PR and https://github.com/SymbiFlow/symbiflow-arch-defs/pull/738 ?  If this PR was simply adding annotations that #738 then made concrete from SDF in a later step, I'd understand.  But this is directly emitting the values.

Reply:
@litghost You are right, those two approaches are rather contradictory. This PR relates to the issue https://github.com/SymbiFlow/symbiflow-arch-defs/issues/749 by @mithro. However, the code can be easily modified to inject annotations instead of concrete values like in #738

Reply:
I think #738 has the right idea to allow one pb_type (or v2x) to describe both the SLICEM and SLICEL variants (or ALUT, BLUT, etc) without the need to repeat the sim verilog.  I like the idea of auto-annotationing v2x, but let's emit the SDF tag for #738 to convert to a concrete value.  This will have additional value when do corner analysis or handling speed grades.

Reply:
Ok, So maybe I'll make the v2x output annotations by default and add a switch which will make it output values. Concrete timing values in a Verilog model might be useful eg. for simulation.